app/concepts/api/v1/users/registrations/operation/create.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Api::V1::Users::Registrations::Operation
+  class Create < ApplicationOperation
+    step :validate
+    step :create
+
+    private
+
+    def validate(params)
+      result = Api::V1::Users::Registrations::Contract::Validate.call(params)
+      result.errors.to_h.empty? ? Success(params) : Failure(result)
```

Comment:
Are you shure that `result.errors` isn't enumerable? If yes, you can use predicate `.empty?` direcrtly.

Reply:
I agree. Will change to direct usage.